streampark-console/streampark-console-service/src/main/java/org/apache/streampark/console/core/controller/SavePointController.java

```diff
@@ -64,4 +66,11 @@ public RestResponse delete(Long id) throws InternalException {
     Boolean deleted = savePointService.delete(id, application);
     return RestResponse.success(deleted);
   }
+
+  @PostMapping("trigger")
+  @RequiresPermissions("savepoint:trigger")
+  public RestResponse trigger(Long appId, @Nullable String savepointPath) {
```

Comment:
Could you add swagger annotations to help third-party services interface with streampark?

Reply:
done. thank you.